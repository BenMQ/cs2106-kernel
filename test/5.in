cr a 2
cr b 2
cr c 2
cr d 2
to
to
to
to
req R1 1
req R2 3

init
cr a 2
cr b 2
cr c 2
to
cr j 1
req R2 2
to
cr e 1
cr d 1
req R2 1
req R1 1
to
req R1 1
req R2 1
req R3 3
req R2 1
req R2 1
req R2 1
de b
to
de f
