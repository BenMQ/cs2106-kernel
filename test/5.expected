init a a a a b c d a a error
init a a a b b b c c c a a b a j j e d init c a error
